### PR TITLE
[EN] Corrections.

### PR DESCRIPTION
**Pull request by ignotus666**
I'm making a pull request to the 'changes' branch as many of the edits need to be included in the translations (e.g. corrected website addresses). Many corrections are just of typos, but in other instances sentences have been changed where I thought they were too vague - though suggestions are welcome.

### CODE REVIEW

**Comment by trebmuh** [comment]
LGTM

File: wiki/en/en-Client-Troubleshooting.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Start with the simple stuff: make sure your instrument/microphone and headphones
 
 You can test whether you are hearing your signal correctly by doing the following:
 
-1. Get your audio into your computer.
+1. Connect your audio source (mic, instrument...) to your computer.
```
Comment (ann0see) [inline wiki/en/en-Client-Troubleshooting.md]: remark-lint reported some issues here. I think we might need to go through the whole wiki and fix syntax errors. 
That's nothing you should need to worry about now.

File: wiki/en/en-Getting-Started.md [hunk 2]
```diff
@@ -11,21 +11,21 @@ permalink: "/wiki/Getting-Started"
 
 1. **A computer** running either **Windows 10**, **macOS** (**MacOS X** El Capitan v10.11 or higher), or **Linux**.
 1. **A wired internet connection** (don't use Wi-Fi, it will cause sound problems). Most broadband connections will be fine[^1].
-1. **Wired headphones - not speakers** to prevent echo or feedback and to minimize delay[^2].
+1. **Wired headphones - not speakers** to prevent echo or feedback and to minimise delay[^2].
```
Comment (ann0see) [inline wiki/en/en-Getting-Started.md:14]: I think we already had a discussion about BE/AE and I think minimize is also ok in BE?

File: wiki/en/en-Hardware-Setup.md [hunk 3]
```diff
@@ -25,14 +25,14 @@ Setting up hardware usually burns down to 4 points, although each setup is diffe
 
 ## Windows: Audio interface connection - ASIO4ALL
 
-This is an example Windows client installation with audio device [Behringer U-CONTROL UCA202](https://www.amazon.com/Behringer-U-Phono-UFO202-Audiophile-Interface/dp/B002GHBYZ0).
+This is an example Windows client installation with the [Behringer U-CONTROL UCA202](https://www.amazon.com/Behringer-U-Phono-UFO202-Audiophile-Interface/dp/B002GHBYZ0) audio device.
 The following instructions might be similar with other audio devices.
```
Comment (ann0see) [inline wiki/en/en-Hardware-Setup.md:31]: `audio device`. Which brings us back to the question if we should use interface/audio decice/sound card,... Not sure about the outcome.

File: wiki/en/en-Installation-for-Linux.md [hunk 4]
```diff
@@ -14,18 +14,18 @@ Although we don't provide official packages for every Linux distribution, you mi
...
 
-There is also list of unofficial [packages for different distributions here](https://github.com/corrados/jamulus/issues/223).
+There is also list of unofficial [packages for different distributions here](https://github.com/jamuluswebsite/jamulus/issues/223).
 
 ### Debian and Ubuntu (amd64 .deb only)
 
 If you're on amd64 **Debian**/**Ubuntu**, you may try the compiled .deb packages from GitHub Actions.
 
-1. Download the jamulus_latest_ubuntu_amd64.deb file from the [Jamulus GitHub release](https://github.com/corrados/jamulus/releases/tag/latest) page
+1. Download the jamulus_latest_ubuntu_amd64.deb file from the [Jamulus GitHub release](https://github.com/jamuluswebsite/jamulus/releases/tag/latest) page
```
Comment (ann0see) [inline wiki/en/en-Installation-for-Linux.md]: We'll probably need to update this link once the release is made. Also `apt --fix-broken install` is needed on a clean machine. (Not sure if there's a cleaner way to install the dependencies...)

**Review by ann0see — COMMENTED** [review]
Thanks very much for yor work! This looks great. See my comments.

File: wiki/en/en-Getting-Started.md (hunk 2 above)
Comment (ignotus666) [reply wiki/en/en-Getting-Started.md:14]: Minimise is preferred in BE.

File: wiki/en/en-Client-Troubleshooting.md (hunk 1 above)
Comment (ignotus666) [reply wiki/en/en-Client-Troubleshooting.md]: Yeah, I didn't have a clue what it was on about.

File: wiki/en/en-Hardware-Setup.md (hunk 3 above)
Comment (ignotus666) [reply wiki/en/en-Hardware-Setup.md:31]: Me neither... that's why I left it as-is.

File: wiki/en/en-Central-Servers.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ permalink: "/wiki/Central-Servers"
 
 Because there is a limit of 150 servers per central server, you need to select which central server you want to advertise your server on. Select a genre if you intend to limit players to that style (players can then see your server in their genre list). 
 
-When you start a public server, you will see a message saying whether your server has been successfully listed or not. If not, and you leave your server running, it will re-try its attempts to register until a free slot becomes available.  
+When you start a public server, you will see a message saying whether your server has been successfully listed or not. If not, and you leave your server running, it will reattempt to register until a free slot becomes available.
```
Comment (pljones) [inline wiki/en/en-Central-Servers.md]: "Keep trying"... keep it simple :)

File: wiki/en/en-Contribution.md [hunk 6]
```diff
@@ -9,19 +9,19 @@ permalink: "/wiki/Contribution"
...
 
-See the [Jamulus GitHub repo](https://github.com/corrados/jamulus) for further information.
+See the [Jamulus GitHub repo](https://github.com/jamulussoftware/jamulus) for further information.
 
 ## Jamulus Website
 
-* If you find a mistake, typo or something out of date (in any language) on the website, you can [open an issue on GitHub](https://github.com/jamulussoftware/jamuluswebsite/issues) or, if you don't have a GitHub account on [the relevant discussion forum](https://sourceforge.net/p/llcon/discussion/) on SourceForge.
+* If you find a mistake, typo or something out of date (in any language) on the website, you can [open an issue on GitHub](https://github.com/jamulussoftware/jamuluswebsite/issues) or if you don't have a GitHub account, on [the relevant discussion forum](https://sourceforge.net/p/llcon/discussion/) on SourceForge.
```
Comment (pljones) [inline wiki/en/en-Contribution.md]: There should be a comma before and after `if you don't have a GitHub account`.  Can't remember what part of speech that makes it...

File: wiki/en/en-Hardware-Setup.md [hunk 7]
```diff
@@ -56,24 +56,24 @@ You're done! Have fun!
...
+You might want to install [Ubuntu Studio](https://ubuntustudio.org/). It adds a second option in your boot menu for a low-latency kernel. The key to successful jamming is "low latency" between servers and connected Jamulus clients. If the underlying Linux system is started with a low-latency kernel then it has a positive impact on latency for your Jamulus sessions.
 
 
 ## Points to note about inputs
 
-- If you want to connect 2 or more sources (eg voice + synth + guitar), it is important to note that Jamulus currently handles only input 2 channels (L/R). So the hardware being used must provide a mixed-down stereo output to Jamulus.
+- If you want to connect 2 or more sources (e.g. voice + synth + guitar), it is important to note that Jamulus currently handles only 2 input channels (L/R). So the hardware being used must provide a mixed-down stereo output to Jamulus.
```
Comment (pljones) [inline wiki/en/en-Hardware-Setup.md]: "Alternatively run two instances of Jamulus, if your hardware input supports it.  Or use virtual audio inputs to Jamulus for each hardware input."

I don't mind keeping it simple - but the original is wrong in the sense it states that no other solution is possible.

File: wiki/en/en-Installation-for-Linux.md (hunk 4 above)
Comment (pljones) [reply wiki/en/en-Installation-for-Linux.md]: Ideally we'd just tell people to install from their distribution, rather than from here.  They'll likely feel happier about doing it, too, and once it's installed, it'll keep them up to date fairly easily.

We should advise people that many distributions do provide a current version of Jamulus and they only need build it themselves if either they want to get involved in development or their distribution doesn't supply it (and we'd appreciate volunteers for distributions taking on provision as a way of getting involved).

File: wiki/en/en-Installation-for-Windows.md [hunk 8]
```diff
@@ -28,14 +28,14 @@ Jamulus uses [ASIO](https://en.wikipedia.org/wiki/Audio_Stream_Input/Output) to
 
 For ASIO4ALL ([download](http://www.asio4all.org){: target="_blank" rel="noopener noreferrer"}), you may need to experiment a bit to find the right configuration since every computer is a bit different.
 
-**Tip:** Setup your sound card while you're [connected to a server](Onboarding#2-connecting-to-a-server) to hear your instrument or voice and check if everything is correctly setup, but first read on.
+**Tip:** Set up your sound card while you're [connected to a server](Onboarding#2-connecting-to-a-server) to hear your instrument or voice and check if everything is correctly set up; but first read on.
```
Comment (pljones) [inline wiki/en/en-Installation-for-Windows.md:35]: Tip: install the ASIO4All "Off-line Settings" panel.

File: wiki/en/en-Installation-for-Windows.md [hunk 9]
```diff
@@ -46,15 +46,15 @@ Before you start with Jamulus:
```
Comment (pljones) [inline wiki/en/en-Installation-for-Windows.md:49]: In the above (line 44), why does it only mention inputs?   I think this means "Click the plus icon to open the sound card you want to use to show its inputs and outputs, if they are not shown"

File: wiki/en/en-Installation-for-Windows.md [hunk 10]
```diff
@@ -46,15 +46,15 @@ Before you start with Jamulus:
...
+Afterwards, *set up the inputs/outputs again*. Enabled and accessible input/outputs show as lit up power buttons and play buttons. If instead you see a red cross or yellow symbol, close other applications that may be accessing your sound card (e.g. browser, Zoom, etc).
 
-Have a look at [this video](https://youtu.be/_GzOsitVgLI) from [trombonepizza](https://github.com/trombonepizza) which gives more detailed setup information on ASIO4ALL.
+Have a look at [this video](https://youtu.be/_GzOsitVgLI) by [trombonepizza](https://github.com/trombonepizza) which gives more detailed setup information on ASIO4ALL.
 
 Official and further information about how to configure ASIO4ALL is documented in the official [ASIO4ALL FAQs on the ASIO4ALL website](http://www.asio4all.org/faq.html){: target="_blank" rel="noopener noreferrer"} and in the [ASIO4ALL manual here](https://www.asio4all.org/ASIO4ALL%20v2%20Instruction%20Manual.pdf).
```
Comment (pljones) [inline wiki/en/en-Installation-for-Windows.md]: What is this? `noopener noreferrer`

File: wiki/en/en-Network-Requirements.md [hunk 11]
```diff
@@ -19,7 +19,7 @@ With the following units
 * Kbit/s : Kilo-bits per second (Reminder : 1 Mbit/s = 1024 Kbit/s, 1 KByte/s = 8 Kbit/s)
 * Mbit/s : Mega-bits per second
 
-| Channels  | Quality | Bandwidth (for buffer : 2.67 ms) |  Bandwidth (for buffer : 5.33ms) | Bandwidth (for buffer : 10.67 ms) | Bandwidth (for buffer : 21.33 ms) |
+| Channels  | Quality | Bandwidth (for buffer : 2.67 ms) |  Bandwidth (for buffer : 5.33 ms) | Bandwidth (for buffer : 10.67 ms) | Bandwidth (for buffer : 21.33 ms) |
 | --------- | ------ | -------- | -------- | -------- | -------- |
 | Stereo    | High   | 894 Kbit/s | 657 Kbit/s | 541 Kbit/s | 483 Kbit/s |
```
Comment (pljones) [inline wiki/en/en-Network-Requirements.md:24]: Might be worth checking these - I think I remember they went up.

File: wiki/en/en-Onboarding.md [hunk 12]
```diff
@@ -62,6 +62,6 @@ If you need help or want to contribute to the Jamulus Project, you can have a lo
 For help, chat, and general discussions about Jamulus: Have a look at the [forums on SourceForge](https://sourceforge.net/p/llcon/discussion/).
 
 ### Development and contribution
-If you want to contribute to the Jamulus Software, have a look at the [Jamulus GitHub repository](https://github.com/corrados/jamulus/) and read the contributing file there.
+If you want to contribute to the Jamulus Software, have a look at the [Jamulus GitHub repository](https://github.com/jamuluswebsite/jamulus/) and read the contributing file there.
```
Comment (pljones) [inline wiki/en/en-Onboarding.md]: Highlight the file name (and make sure it's correct).

File: wiki/en/en-Hardware-Setup.md (hunk 3 above)
Comment (gilgongo) [reply wiki/en/en-Hardware-Setup.md:31]: This was my take https://github.com/jamulussoftware/jamuluswebsite/issues/168#issuecomment-743317373 (which would mean that this amend is fine)

**Review by gilgongo — APPROVED** [review]

File: wiki/en/en-Software-Manual.md [hunk 13]
```diff
@@ -246,7 +246,7 @@ Leave this blank unless you need to enter the address of a central server other
 ![Indicators](https://user-images.githubusercontent.com/20726856/97362338-271dd580-18a1-11eb-99b7-02c9371a6258.png)
 
 The Ping Time is the time required for the audio stream to travel from the client to the server and back again.
-This delay is introduced by the network and should about 20-30ms. If this delay is higher than about 50ms,
+This delay is introduced by the network and should be about 20-30 ms. If this delay is higher than about 50 ms,
 your distance to the server is too large or your internet connection is not sufficient.
```
Comment (pljones) [inline wiki/en/en-Software-Manual.md]: I'd rephrase that - it's more than it will be more noticeable and initially harder to stay in time.  People cope with well over 50ms ping happily enough once they're used to it.

File: wiki/en/en-Central-Servers.md (hunk 5 above)
Comment (ignotus666) [reply wiki/en/en-Central-Servers.md]: Yes, it does sound more natural that way. Done.

File: wiki/en/en-Contribution.md (hunk 6 above)
Comment (ignotus666) [reply wiki/en/en-Contribution.md]: Done.

File: wiki/en/en-Hardware-Setup.md (hunk 7 above)
Comment (ignotus666) [reply wiki/en/en-Hardware-Setup.md]: I was mainly sticking to revising existing text - but if the rest of you think that extra info should go in, I'll add it.

Comment (pljones) [reply wiki/en/en-Hardware-Setup.md]: I know what you mean - and I'm not going to hold up this improvement just because it doesn't get us 100% perfection in one hit :).

File: wiki/en/en-Installation-for-Windows.md (hunk 10 above)
Comment (ann0see) [reply wiki/en/en-Installation-for-Windows.md]: Have a look at [this article](https://pointjupiter.com/what-noopener-noreferrer-nofollow-explained/)

File: wiki/en/en-Hardware-Setup.md (hunk 3 above)
Comment (ann0see) [reply wiki/en/en-Hardware-Setup.md:31]: ok. Will resolve this

File: wiki/en/en-Installation-for-Linux.md (hunk 4 above)
Comment (ann0see) [reply wiki/en/en-Installation-for-Linux.md]: Agree. #287 might be something we should include here? Probably the installation on Linux needs to be rewritten.

**Comment by gilgongo** [comment]
Looks like we're revving up for a release so it would be good to resolve whatever it is that's blocking this and get it merged. If we need to keep it to just grammar and spelling then so be it, unless you can take in those other suggestions @ignotus666 ?

File: wiki/en/en-Installation-for-Linux.md (hunk 4 above)
Comment (gilgongo) [reply wiki/en/en-Installation-for-Linux.md]: The Linux installation stuff needs to be re-done now that we're in the repos properly, with compilation instructions relegated to a file in the repo I think (and just linked to in passing from the website). 

But in the interests of time, can we keep as is? Release is coming soon...

**Comment by ignotus666** [comment]
> Looks like we're revving up for a release so it would be good to resolve whatever it is that's blocking this and get it merged. If we need to keep it to just grammar and spelling then so be it, unless you can take in those other suggestions @ignotus666 ?

I can't right now but I'll have a look at it during the day tomorrow (Monday).

**Comment by ann0see** [comment]
I think this has to be squashed and merged ;-).

**Review by pljones — APPROVED** [review]
Apart from the PR title (it should say what I should expect to find changed), I'm happy.

File: wiki/en/en-Hardware-Setup.md (hunk 7 above)
Comment (ignotus666) [reply wiki/en/en-Hardware-Setup.md]: Done.

File: wiki/en/en-Installation-for-Windows.md (hunk 8 above)
Comment (ignotus666) [reply wiki/en/en-Installation-for-Windows.md:35]: I think this needs more of an explanation. I don't use Windows so don't really know what this means/does.

File: wiki/en/en-Installation-for-Windows.md (hunk 9 above)
Comment (ignotus666) [reply wiki/en/en-Installation-for-Windows.md:49]: Done

File: wiki/en/en-Network-Requirements.md (hunk 11 above)
Comment (ignotus666) [reply wiki/en/en-Network-Requirements.md:24]: Yes, I think they've gone up slightly. Also - it's missing the figures for when "Enable small network buffers" _isn't_ checked when using 2.67 ms.

I don't have time to check all these right now. I don't think it'll cause much harm if this doesn't get updated yet though - it's just a few Kb in each case.

File: wiki/en/en-Onboarding.md (hunk 12 above)
Comment (ignotus666) [reply wiki/en/en-Onboarding.md]: Done

File: wiki/en/en-Software-Manual.md (hunk 13 above)
Comment (ignotus666) [reply wiki/en/en-Software-Manual.md]: Rephrased that. Let me know what you think.

File: wiki/en/en-Hardware-Setup.md (hunk 7 above)
Comment (ignotus666) [reply wiki/en/en-Hardware-Setup.md]: Done

**Comment by ignotus666** [comment]
I reckon this should be good to go. I'll update the MIDI controller section when I have a better understanding of the parameters to explain them in layman's terms.

File: wiki/en/en-Onboarding.md [hunk 14]
```diff
@@ -62,6 +62,6 @@ If you need help or want to contribute to the Jamulus Project, you can have a lo
 For help, chat, and general discussions about Jamulus: Have a look at the [forums on SourceForge](https://sourceforge.net/p/llcon/discussion/).
 
 ### Development and contribution
-If you want to contribute to the Jamulus Software, have a look at the [Jamulus GitHub repository](https://github.com/jamuluswebsite/jamulus/) and read the contributing file there.
+If you want to contribute to the Jamulus Software, have a look at the [Jamulus GitHub repository](https://github.com/jamulussoftware/jamulus/) and read the `CONTRIBUTING.MD` file there.
```
Comment (gilgongo) [inline wiki/en/en-Onboarding.md]: CONTRIBUTING.MD or CONTRIBUTING.md?

Comment (ignotus666) [reply wiki/en/en-Onboarding.md]: Lower case .md. I'd left caps on but corrected it.

**Review by gilgongo — APPROVED** [review]

File: wiki/en/en-Client-Troubleshooting.md [hunk 15]
```diff
@@ -67,8 +67,8 @@ See [this forum discussion](https://sourceforge.net/p/llcon/discussion/533517/th
 (With thanks to [Mark Anthony De Souza](https://www.facebook.com/groups/619274602254947/permalink/765122847670121/?comment_id=765525034296569))
 
 Maybe you did not answer "Yes" to the `"Jamulus wants to access your microphone"` challenge.  To fix this:
-* Go to `Preferences` > `Security & Privacy` > `Privacy` tab
-* Find `Microphone` on the left and then make sure `Jamulus` is enabled on the right-hand list
+*  Go to `Preferences` > `Security & Privacy` > `Privacy` tab
```
Comment (pljones) [inline wiki/en/en-Client-Troubleshooting.md]: Revert the space here and below - stupid linter.

File: wiki/en/en-Contribution.md [hunk 16]
```diff
@@ -9,19 +9,19 @@ permalink: "/wiki/Contribution"
...
 
-See the [Jamulus GitHub repo](https://github.com/corrados/jamulus) for further information.
+See the [Jamulus GitHub repo](https://github.com/jamulussoftware/jamulus) for further information.
 
 ## Jamulus Website
 
-* If you find a mistake, typo or something out of date (in any language) on the website, you can [open an issue on GitHub](https://github.com/jamulussoftware/jamuluswebsite/issues) or, if you don't have a GitHub account on [the relevant discussion forum](https://sourceforge.net/p/llcon/discussion/) on SourceForge.
+*  If you find a mistake, typo or something out of date (in any language) on the website, you can [open an issue on GitHub](https://github.com/jamulussoftware/jamuluswebsite/issues) or, if you don't have a GitHub account, on [the relevant discussion forum](https://sourceforge.net/p/llcon/discussion/) on SourceForge.
```
Comment (pljones) [inline wiki/en/en-Contribution.md]: The space doesn't help...

File: wiki/en/en-Installation-for-Linux.md [hunk 17]
```diff
@@ -91,14 +91,14 @@ You can now delete the sources directory you compiled from.
 ### Configure JACK with QjackCtl
 Jamulus clients need [JACK](https://jackaudio.org/) to run, but you need to configure that first. The recommended method is to use `QjackCtl`.
 1. Open the command shell e.g. with Ctrl-Alt-T and
-1. Execute the command `qjackctl` you will see the **JACK Audio Connection Kit**
+1. Execute the command `qjackctl`. You will see the **JACK Audio Connection Kit**
```
Comment (pljones) [inline wiki/en/en-Installation-for-Linux.md]: Hm.  No, you won't.  You'll see the Qt JACK Control utility main page.

File: wiki/en/en-Installation-for-Windows.md (hunk 8 above)
Comment (pljones) [reply wiki/en/en-Installation-for-Windows.md:35]: It's part of the ASIO4ALL installer options.  It means you don't need to run an ASIO application to get into the ASIO4ALL settings - you can do that first.

File: wiki/en/en-Server-Troubleshooting.md [hunk 18]
```diff
@@ -37,7 +37,7 @@ Yes. Please see [Server Types](Choosing-a-Server-Type)
 
 ### Can I convert my private server to being a public one?
 
-Yes. Simply check the "Make my Server Public" box in the server window and fill out the details. Remember that you also do not need to port forward if you have a public server.
+Yes. Simply check the "Make my Server Public" box in the server window and fill out the details. Remember also that you do not need to port forward if you have a public server.
```
Comment (pljones) [inline wiki/en/en-Server-Troubleshooting.md]: I'd rephrase to say "If you register your server, do not use port forwarding as it may prevent others using your server."

File: wiki/en/en-Server-Win-Mac.md [hunk 19]
```diff
@@ -24,13 +24,13 @@ permalink: "/wiki/Server-Win-Mac"
 
 1. Type in the name, city and country so that other users can easily identify you.
 
-1. Specify a musical genre if you wish. This will make your server appear in the relevant server list for clients (see more [details on Central Servers](Central-Servers)).
+1. Specify a music genre if you wish. This will make your server appear in the relevant server list for clients (see more [details on Central Servers](Central-Servers)).
```
Comment (pljones) [inline wiki/en/en-Server-Win-Mac.md]: I'd remove "if you wish".  "All Genres 1" is full and anyone should be looking at the server lists (using Jamulus Explorer) to find space and an appropriate genre.

File: wiki/en/en-Software-Manual.md [hunk 20]
```diff
@@ -72,7 +72,7 @@ pressing this button will end the session.
 
 ![Connect dialog](https://user-images.githubusercontent.com/20726856/97360965-31d76b00-189f-11eb-870b-f58be6e30f04.png)
 
-The Connection Setup window shows a list of available servers together with the number of occupants and the maximum number supported. Server operators can optionally list their servers by musical genre. Use the List dropdown to select a genre, click on the server you want to join and press the Connect button to connect to it. Alternatively, double-click on the server name. Permanent servers (those that have been listed for longer than 24 hours) are shown in bold.
+The Connection Setup window shows a list of available servers together with the number of occupants and the maximum number supported. Server operators can optionally list their servers by music genre. Use the List dropdown to select a genre, click on the server you want to join and press the Connect button to connect to it. Alternatively, double-click on the server name. Permanent servers (those that have been listed for longer than 24 hours) are shown in bold.
```
Comment (pljones) [inline wiki/en/en-Software-Manual.md]: It's not optional.  All registered servers have a server list.  Some of the server lists are genre-specific.  There could be location-specific ones.  But they're all on lists.  I think this needs explaining more clearly.

**Comment by pljones** [comment]
Getting there - I never see everything in one pass when reviewing...

File: wiki/en/en-Client-Troubleshooting.md (hunk 15 above)
Comment (ignotus666) [reply wiki/en/en-Client-Troubleshooting.md]: Done

File: wiki/en/en-Contribution.md (hunk 16 above)
Comment (ignotus666) [reply wiki/en/en-Contribution.md]: Done

File: wiki/en/en-Installation-for-Linux.md (hunk 17 above)
Comment (ignotus666) [reply wiki/en/en-Installation-for-Linux.md]: Done

File: wiki/en/en-Server-Troubleshooting.md (hunk 18 above)
Comment (ignotus666) [reply wiki/en/en-Server-Troubleshooting.md]: Done

File: wiki/en/en-Server-Win-Mac.md (hunk 19 above)
Comment (ignotus666) [reply wiki/en/en-Server-Win-Mac.md]: Done

File: wiki/en/en-Software-Manual.md (hunk 20 above)
Comment (gilgongo) [reply wiki/en/en-Software-Manual.md]: This is the client's manual don't forget, so we're talking about their view of that list. But maybe we can just drop that line so it reads:

"... the maximum number supported. Use the List dropdown to select a genre, click on the server..."

Comment (ignotus666) [reply wiki/en/en-Software-Manual.md]: Rephrased it. See what you reckon.

File: wiki/en/en-Client-Troubleshooting.md [hunk 21]
```diff
@@ -18,11 +18,10 @@ Start with the simple stuff: make sure your instrument/microphone and headphones
 
 You can test whether you are hearing your signal correctly by doing the following:
 
-1.  Connect your audio source (mic, instrument...) to your computer.
-1.  Use an audio recording application (such as [Audacity](https://www.audacityteam.org/)) on your computer to record that input and check it's OK.
-1.  Make sure that if you mute your audio input inside the recording application, you can't hear yourself.
-1.  Close the recording application and run Jamulus. You should still not hear yourself.
-1.  Connect to a server and play something. Now you should hear yourself with the delay from the server.
+1. Run Jamulus and connect to a server with a long ping time (greater than 200ms)
+2. Clap your hands once (or play one short note on an instrument). You should hear the sound from the Jamulus server come back, but significantly delayed.
```
Comment (pljones) [inline wiki/en/en-Client-Troubleshooting.md]: Oh dear.  So it doesn't understand markdown.  That's not helpful.  At least it's stopped moaning about spaces..!  (I assume it wanted each "1" indented under the previous - makes some sense if it didn't understand they'd auto-number.)

File: wiki/en/en-Installation-for-Windows.md (hunk 8 above)
Comment (ignotus666) [reply wiki/en/en-Installation-for-Windows.md:35]: If it's part of the installer options then I think we should either mention it further up, in the "Download and install an ASIO driver" section, or get rid of that first link to it (it's repeated again -and again- a few lines below; I don't see the point in linking to it all over the place on the same page), and point this option out when the link is provided. Otherwise people might just click on the first link they see and install it, only to later have to uninstall/reinstall it.

**Comment by gilgongo** [comment]
So shall I give this a merge?

File: wiki/en/en-Tips-Tricks-More.md [hunk 22]
```diff
@@ -96,7 +96,7 @@ jack_connect Jamulus:'output right' system:playback_2
...
+The volume fader, pan control and mute & solo buttons in the client's mixer window strips can be controlled using a MIDI controller by using the `--ctrlmidich` parameter (note: only available for use with macOS and Linux using Jamulus version 3.7.0 or higher). To enable this feature, Jamulus must be launched with `--ctrlmidich`. There is one global MIDI channel parameter (1-16) and two parameters you can set for each item controlled: `offset` and `consecutive CC numbers`. Set the first parameter to the channel you want Jamulus to listen on (0 for all channels) and then specify the items you want to control (f = volume fader; p = pan; m = mute; s = solo) with the offset (CC number to start from) and number of consecutive CC numbers. Take the following example:
```
Comment (pljones) [inline wiki/en/en-Tips-Tricks-More.md]: Can we say "and" rather than using the symbol "&" when writing in English, please.

Comment (ignotus666) [reply wiki/en/en-Tips-Tricks-More.md]: Done

**Review by pljones — APPROVED** [review]

**Comment by ann0see** [comment]
Any updates here? Ready to merge?

**Review by pljones — APPROVED** [review]

**Comment by ann0see** [comment]
I will merge this now.
